ttt-game/src/components/player-two.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const PlayerTwo = () => {
```

Comment:
This is kind of cool - means that you can change the player value if needed relatively easily.